### PR TITLE
STYLE-45 loading-pill component source, added some usage details to r…

### PR DESCRIPTION
**Pull request by hrag**
Added loading pill component to index, can now be easily imported!

Also added some usage info in the docs.

### CODE REVIEW

File: src/lib/index.jsx [hunk 1]
```diff
@@ -1 +1,4 @@
 import cards from './cards';
+import LoadingPill from './loading-pill';
+
+export default LoadingPill;
```
Comment (chalisegrogan) [inline src/lib/index.jsx]: FYI, this little icon screen shot 2018-10-15 at 4 06 33 pm indicates that there's no new line at the end of the file. Most linters will pick up on this and you can configure most IDEs to add them on save, but github has an additional icon to point it out. 

Also, this filetype can be `.js` if there's no actual JSX code within the file, even if it's importing files with JSX.

File: src/docs/sections/loading.jsx [hunk 2]
```diff
@@ -431,6 +421,28 @@ stop($('.loading-example-pill1'), () => {
                             Some content
                         </div>
                     </div>
+
+                    <hr />
+                    <h2>React Loading Animations</h2>
```
Comment (chalisegrogan) [inline src/docs/sections/loading.jsx]: I see what you did there 😄

Comment (hrag) [reply src/docs/sections/loading.jsx]: ;)

File: src/lib/index.jsx (hunk 1 above)
Comment (hrag) [reply src/lib/index.jsx]: Oh yeah, true.

And yeah I thought my linter was configured for the trailing line. Will check that.

File: src/docs/sections/loading.jsx (hunk 2 above)
Comment (antfitch) [reply src/docs/sections/loading.jsx]: So cool! I see one thing that I think should change. Checkout the difference between how the import for JS and JSX is documented. My vote is to nudge the JSX import to look like the JS one. 

screen shot 2018-10-15 at 9 21 35 pm

screen shot 2018-10-15 at 9 21 43 pm

Comment (blopker) [reply src/docs/sections/loading.jsx]: Yes, we should try and namespace as much as possible.

**Pull request merged**